isis/tests/data/defaultImage/defaultCube.isd

```diff
@@ -1,83 +1,146 @@
-{"CameraVersion": 2,
-   "NaifKeywords": {
-        "BODY_CODE" : 499,
-        "BODY499_RADII" : [3396.19, 3396.19, 3376.2],
-        "BODY_FRAME_CODE" : 10014,
-        "CLOCK_ET_-27999_33322515_COMPUTED" : "d5b9203a4d24c5c1",
-        "INS-27002_TRANSX" : [0.0, 0.011764705882353, 0.0],
-        "INS-27002_TRANSY" : [0.0, 0.0, 0.01176470588235],
-        "INS-27002_ITRANSS" : [0.0, 85.0, 0.0],
-        "INS-27002_ITRANSL" : [0.0, 0.0, 85.0]
+ {
```

Comment:
This concerns me but I understand why we need to have a real ISD for anything that needs a camera.